api/src/resources/dataset.rs

```diff
@@ -34,6 +34,7 @@ pub struct Dataset {
     pub entity_defs: Vec<EntityDef>,
     pub label_defs: Vec<LabelDef>,
     pub label_groups: Vec<LabelGroup>,
+    pub num_reviewed: Option<f64>,
```

Comment:
Do you need something like a `#[serde(default)]` here, as `num_reviewed` is not actually returned by the datasets api endpoint?

This is kind of breaking the encapsulation of this type a bit - as it lives in `api`, I would expect it to map to our api types.

I think it would be better practice to have a struct like
```rust
struct DatasetAndStats {
    dataset: Dataset,
    stats: Option<DatasetStats>
}
```

and you can implement `DisplayTable` for that instead. Or, just have a type in the `cli` named `Dataset` that you can convert `From` the `api` Dataset, and add some more stats to.

If you decide not to do the above, I think you need to add a comment documenting that this field is not returned by the datasets api

Reply:
Yep, great point. I'll update